src/raps/unlockAndCrosschainSwap.ts

```diff
@@ -71,7 +69,14 @@ export const estimateUnlockAndCrosschainSwap = async ({
     quote,
   });
 
+  if (swapGasLimit === null || swapGasLimit === undefined || isNaN(Number(swapGasLimit))) {
+    return null;
```

Comment:
this will bubble up to the estimation and use the `basic_swap` defaults for the chain